src/recon.rs

```diff
@@ -460,7 +463,7 @@ unsafe fn decode_coefs<BD: BitDepth>(
     let frame_hdr = &***(*f).frame_hdr.as_ref().unwrap();
     let lossless = frame_hdr.segmentation.lossless[b.seg_id as usize];
     let t_dim = &dav1d_txfm_dimensions[tx as usize];
-    let dbg = DEBUG_BLOCK_INFO(&*f, &*t) as c_int;
+    let dbg = debug_block_info!(&*f, &*t) as c_int;
```

Comment:
```suggestion
    let dbg = (debug_block_info!(&*f, &*t) && plane != 0 && false) as c_int;
```

Separate issue, but this seems mistranslated looking at the C source.

Reply:
Fixed in f3eae29b224a8a86629bc78f020c1746895f2eed